numpy/add_newdocs.py

```diff
@@ -547,10 +547,26 @@ def luf(lamdaexpr, *args, **kwargs):
 
     """))
 
-add_newdoc('numpy.core', 'broadcast', ('nd',
+add_newdoc('numpy.core', 'broadcast', ('ndim',
     """
     Number of dimensions of broadcasted result.
 
+    .. versionadded:: 1.12.0
+
```

Comment:
See below - I reversed the direction of the alias, to encourage use of `ndim` over `nd`. Is this undesirable?